src/Domain/Entities/UserChat.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CoduTeam.Domain.Entities;
+
+public class UserChat
```

Comment:
Make it a child of BaseAuditableEntity 
see https://github.com/dimanoit/CoduTeam/blob/main/src/Domain/Entities/UserProject.cs